azerothcore/azerothcore-wotlk
# fix(docker/scripts) add quotes to pwd in docker build scripts

## CHANGES PROPOSED:
-  add quotes to pwd in docker build scripts, This fixes the docker build from failing when you have a space in your directory path

before pr:
![image](https://user-images.githubusercontent.com/4015312/99180007-663c8b00-26d7-11eb-8ab8-706e3d24ac6b.png)

after pr:
![image](https://user-images.githubusercontent.com/4015312/99180095-3f328900-26d8-11eb-8f80-ccca0632cee3.png)

## ISSUES ADDRESSED:
- Closes none



## TESTS PERFORMED:
- create test directory with a space in it
- clone the repo to the directory
- try running the docker build scripts `./bin/acore-docker-build` and `./bin/acore-docker-build-no-scripts` (only tested on Centos7)
- verified that the build completed with no errors



## HOW TO TEST THE CHANGES:

- create test directory with a space in it and cd into it
- clone the repo to the directory
- follow the  pr testing directions to download the changes https://www.azerothcore.org/wiki/How-to-test-a-PR
- try running the docker build scripts `./bin/acore-docker-build` and `./bin/acore-docker-build-no-scripts` (only tested on Centos7)
- verify that the build completed with no errors

## KNOWN ISSUES AND TODO LIST:

- [x] test on macos


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
